contrib/zbm-sign.pl

```diff
@@ -20,6 +20,8 @@
 #       DeleteUnsigned: false
 #       SignMethod: sbctl
 #       KeyDir: /etc/sbkeys
+#       KeyFileName: DB.key
+#       CrtFileName: DB.crt
```

Comment:
The purpose of `KeyDir` is to provide a single configuration option that provides an access to both the cert and key. Adding separate name fields for each now requires setting three options to refer to two values. This is bad UX.

Either keep the `KeyDir` as is, or require `KeyFile` and `CertFile` as independent, full paths.